### ISSUE TITLE
Activating siren causes Wyze Cam V3 to crash

### ISSUE THREAD

**Issue by teixeluis** [mrlt8/docker-wyze-bridge#1051]
Hello,

Shortly after updating  to docker-wyze-bridge v2.5.2 I went ahead and started testing the new entities created in home assistant. I have two cameras (Wyze Cam V3) running fw 4.36.11.5859.

I tested the infrared and night vision switches without problems, and then tested the siren in one of the cameras. Siren played, and camera continued operating normally. When I tried doing the same for the second camera, I immediately lost contact with it once the siren was activated, never to gain access again. The cameras are in a remote site, and currently I have no way of power cycling these.

The cameras are not physically far from each other. On the camera that is still alive I cannot hear the siren from the other one, which suggests that at least it did not hang with the siren running in a loop.

Can there be some issue with the siren command, causing another behaviour in the camera (e.g. resulting in a shutdown instead of playing the siren)?

When I switched it on, the siren entity toggled to on:

![image](https://github.com/mrlt8/docker-wyze-bridge/assets/51710382/e0b8e36d-d9b6-4a4c-9871-eff40c658894)

Does that suggest that the camera still acknowledged the siren being turned on, prior to going offline?

The state of the entity remains the same in spite of the camera being offline for almost 24 hours.

Thanks

**Comment by teixeluis** [comment]
Meanwhile, as of this writing, the camera is available again after almost 24 hours. Perhaps an internal watchdog kicked in and restarted the camera?

**Comment by mrlt8** [comment]
Thanks for the logs, I'll try to see what's causing the error.

Can you replicate the error? Does restarting the bridge help?

I still need to update the siren status as it will shut off after 30 but the status doesn't update:  #953

**Comment by teixeluis** [comment]
Yes, repeated the test and the problem occurred again. I kept listening on the other camera and could confirm that the alarm went on. This time the entity did not change state.

Not much information in the logs this time. Apparently just the result of the camera going offline:

```
2023-11-27 10:29:48.950 ERROR (stream_worker) [homeassistant.components.stream.stream.camera.west_cam] Error from stream worker: Stream ended; no additional packets
2023-11-27 10:29:58.957 ERROR (stream_worker) [homeassistant.components.stream.stream.camera.west_cam] Error from stream worker: Error opening stream (HTTP_NOT_FOUND, Server returned 404 Not Found) rtsp://192.168.1.64:8554/ovalesublime-west-cam
2023-11-27 10:30:18.964 ERROR (stream_worker) [homeassistant.components.stream.stream.camera.west_cam] Error from stream worker: Error opening stream (HTTP_NOT_FOUND, Server returned 404 Not Found) rtsp://192.168.1.64:8554/ovalesublime-west-cam
```

**Comment by mrlt8** [comment]
Is the lost connection happening with just the one camera? Does the same thing happen in the wyze app? e.g. turning on the siren drops the video on the wyze app? 

I'm wondering if its an issue with the camera and/or power supply for that one camera?

**Comment by teixeluis** [comment]
@mrlt8 yes so far just with that one camera in particular. The first time I tested the feature was with its sibling camera which didn't crash, but I did not repeat the test on that one because I need it alive. Once the camera (hopefully) reboots tomorrow I will repeat the test with the app and see if the same happens. I don't remember this happening in the past with the app.

In terms of power the two cameras share the same cabling. On the other end I have two USB wall adapters wired in parallel to provide more power for both cameras.

**Comment by mrlt8** [comment]
Is it working in the app? Can you try rebooting the camera via the app or via the webUI or api `:5000/api/ovalesublime-west-cam/restart`?

**Comment by yeahme49** [comment]
I used to have this issue with one of my V3 cameras. The issue was I had a couple USB extension cables connected between the camera and the power source, which caused the voltage to drop. When activating the siren the additional current dropped the voltage even further and the camera would crash.

I solved this by switching from a USB power adapter to a 12v power adapter and then using https://www.amazon.com/gp/product/B08H89LTP5/ at the camera to convert the 12v to 5v USB.
